template-only-docs/Deployment.md

```diff
@@ -0,0 +1,43 @@
+# Deployment to cloud environments
+
+## Requirements
+
+- External service access: The application must be able to make network calls to the public internet.
+- AWS Cognito: The application comes with AWS Cognito enabled by default, so AWS Cognito must be set up before deploying the application.
```

Comment:
AWS Cognito has a lot of settings/defaults, and it's not clear from this how to set up a cognito user pool to satisfy the application. A terrafrom module would go a long way to describing what an example cognito/user pool config should look like.

Reply:
Agreed. I'll make a note.